pages/myArticles.js

```diff
@@ -15,11 +17,22 @@ function MyArticles({ articles }) {
           key="title"
         />
       </Head>
-
-      <ArticleList
-        articles={authorArticles}
-        secondHeadline="Das sind deine Artikel:"
-      />
+      {authorArticles.length !== 0 ? (
+        <ArticleList
+          articles={authorArticles}
+          secondHeadline="Das sind deine Artikel:"
+        />
+      ) : (
+        <h2>
+          Du hast keine Artikel zum Verkauf.{" "}
+          <StyledLink
+            href="articleSell"
+            variant="linkToSell"
+          >
+            Hier kannst du deine Artikel erstellen
+          </StyledLink>
+        </h2>
+      )}
```

Comment:
Just a nitpicking suggestion 🤓

```suggestion
      {authorArticles.length  ? (
        <ArticleList
          articles={authorArticles}
          secondHeadline="Das sind deine Artikel:"
        />
      ) : (
        <h2>
          Du hast keine Artikel zum Verkauf.
          <StyledLink
            href="articleSell"
            variant="linkToSell"
          >
            Hier kannst du deine Artikel erstellen
          </StyledLink>
        </h2>
      )}
```

Reply:
thanks Jessica. As always top feedback